examples/ChatQnA/deploy/gaudi.md

```diff
@@ -240,7 +241,6 @@ Check if you have the below set of docker images, before moving on to the next s
 * opea/chatqna-ui:latest

```

Comment:
We did a tagged github checkout and here we use latest images as opposed to a tagged image? Would be nice if we could pass in a tag for the images into the docker file.

Reply:
This can be done. The version will just have to be updated every time we have a new release

Reply:
Lets as @chensuyue. It would be wonderful if we could have reproducibility .. for those not working on dev/latest. All our releases will be tagged. And if that tag matches the checkout, it would be perfect. In the compose file if we used $tag with it defaulting to latest. Even doing it just for ChatQnA now it would be great given this is our prime genAI pipeline. We can do others later.

Reply:
@alexsin368 please address DCO and doc build issue